src/components/TimerPanel.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import classnames from 'classnames';
+
+const TimerPanel = ({ numberOfMinutes, numberOfSeconds, sessionNumber, markSessionAsComplete }) => {
+  // belows are local states(to this file), will only be used here, not global
+  const [minutes, setMinutes] = useState(numberOfMinutes);
+  const [seconds, setSeconds] = useState(numberOfSeconds);
+
+  // the 1st session never starts on its own
+  // if its is the end of all sessions, we go back to display the 1st session, without auto-starting
+  const checkIfLastSession = () => {
+    return sessionNumber > 0 ? true : false;
+  }
```

Comment:
The function name doesn't fit the code. Also you can just return the compare statement.
```suggestion
  const isFirstSession = () => {
    return sessionNumber === 0;
  }
```